docusaurus.config.js

```diff
@@ -192,6 +192,10 @@ const config = {
                 label: 'LinkedIn',
                 href: 'https://linkedin.com/company/uptane',
               },
+              {
+                label: 'Youtube',
+                href: 'https://www.youtube.com/@Uptane',
+              }
```

Comment:
```suggestion
              },
```
I think we want the trailing comma based on best practices.